### PR TITLE
chore(publish): Publish flame_test 1.1.0

### PR DESCRIPTION
**Pull request by spydon**
# Description

Publish new version of flame_test

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the
relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have read the [Contributor Guide] and followed the process outlined for submitting PRs.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples`.

## Breaking Change

Does your PR require Flame users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change. (Indicate it in the [Conventional Commit] prefix with a `!`,
  e.g. `feat!:`, `fix!:`).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flame-engine/flame/issues
[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Flame Style Guide]: https://github.com/flame-engine/flame/blob/main/STYLEGUIDE.md
[Conventional Commit]: https://conventionalcommits.org

### CODE REVIEW

File: packages/flame_test/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# CHANGELOG
+## 1.1.0
+
+ - Add `closeToVector` matcher.
+
+# OLD CHANGELOG
```
Comment (st-pasha) [inline packages/flame_test/CHANGELOG.md:5]: Is this line really necessary? I'd rather remove the square brackets around each header below, to make it all look harmonious.

File: packages/flame_test/example/pubspec.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  flame_test: ^1.0.0
+  flame_test: ^1.1.0
   test: ^1.17.10
```
Comment (st-pasha) [inline packages/flame_test/example/pubspec.yaml:20]: This is incompatible with latest `test` version 1.20.1

Comment (spydon) [reply packages/flame_test/example/pubspec.yaml:20]: But we are not depending on the latest version, so it should be able to resolve an older version, or what do you mean?

Comment (st-pasha) [reply packages/flame_test/example/pubspec.yaml:20]: Nvm, seems like it's `flutter_test`'s problem: they depend on a very specific version of `test_api`, which makes them incompatible with the latest version of `test`.

**Review by wolfenrain — APPROVED** [review]

**Pull request merged**